src/atomate2/abinit/schemas/calculation.py

```diff
@@ -0,0 +1,378 @@
+            - "line": Parse the bandstructure as a line mode calculation with
+              projections
+            - True: Parse the bandstructure as a uniform calculation with
+              projections .
+            - False: Parse the band structure without projects and just store
+              vbm, cbm, band_gap, is_metal and efermi rather than the full
+              band structure object.
+
+        store_trajectory: bool
+            Whether to store the ionic steps as a pmg trajectory object, which can be
+            pushed, to a bson data store, instead of as a list od dicts. Useful for
+            large trajectories.
+        store_scf: bool
+            Whether to store the SCF convergence data.
+        #store_volumetric_data: Sequence[str] or None
+            #Which volumetric files to store.
+
+        Returns
+        -------
+        .Calculation
+            An Abinit calculation document.
+        """
+        dir_name = Path(dir_name)
+        abinit_gsr_file = dir_name / abinit_gsr_file
+        abinit_log_file = dir_name / abinit_log_file
+        abinit_abort_file = dir_name / abinit_abort_file
+
+        # volumetric_files = [] if volumetric_files is None else volumetric_files
+        abinit_gsr = GsrFile.from_file(abinit_gsr_file)
+
+        completed_at = str(datetime.fromtimestamp(os.stat(abinit_log_file).st_mtime))
+
+        # output_file_paths = _get_output_file_paths(volumetric_files)
+        # abinit_objects: dict[AbinitObject, Any] = _get_volumetric_data(
+        #    dir_name, output_file_paths, store_volumetric_data
+        # )
+
+        # dos = _parse_dos(parse_dos, abinit_output)
+        # if dos is not None:
+        #    abinit_objects[AbinitObject.DOS] = dos  # type: ignore
+
+        # bandstructure = _parse_bandstructure(parse_bandstructure, abinit_output)
+        # if bandstructure is not None:
+        #    abinit_objects[AbinitObject.BANDSTRUCTURE] = bandstructure  # type: ignore
+
+        output_doc = CalculationOutput.from_abinit_gsr(abinit_gsr)
+
+        report = None
+        has_abinit_completed = TaskState.FAILED
+        # TODO: How to detect which status it has here ?
+        #  UNCONVERGED would be for scf/nscf/relax when it's not yet converged
+        #  FAILED should be for a job that failed for other reasons.
+        #  What about a job that has been killed by the run_abinit (i.e. before
+        #  Slurm or PBS kills it) ?
+
+        try:
+            report = get_event_report(
+                ofile=File(abinit_log_file), mpiabort_file=File(abinit_abort_file)
+            )
+            if report.run_completed:
+                has_abinit_completed = TaskState.SUCCESS
+            critical_events_report = report.filter_types(critical_events)
+            if critical_events_report:
+                for warning in critical_events_report.warnings:
+                    if type(warning) in AbiConvergenceWarning.all:
```

Comment:
I think this is incorrect. The uncorverged state should only be set if the warning is in the list of `CRITICAL_EVENTS` of each Job. That is the reason for defining the attribute. 
To give an explicit example. You may have a warning for an SCF loop not convereged during a relaxation, but that one should not trigger the uncoverged state, if in the end the relaxation converged.